src/schedule-and-details/requirements-section/entrance-exam/index.jsx

```diff
@@ -87,7 +87,7 @@ EntranceExam.propTypes = {
   isCheckedString: PropTypes.string,
   entranceExamMinimumScorePct: PropTypes.string,
   onChange: PropTypes.func.isRequired,
-  isEditable: PropTypes.bool,
+  isEditable: PropTypes.bool.isRequired,
```

Comment:
You can remove `isEditable` from the `defaultProps` section if it is now required.